Modules/gcmodule.c

```diff
@@ -1124,7 +1124,7 @@ invoke_gc_callback(const char *phase, int generation,
 static Py_ssize_t
 collect_with_callback(int generation)
 {
-    Py_ssize_t result, collected, uncollectable;
+    Py_ssize_t result = 0, collected = 0, uncollectable = 0;
```

Comment:
Nothing. It's only silencing a gcc warning. I can make it a separate PR if you like.